src/model/BankClient.java

```diff
@@ -28,7 +30,24 @@ public void setDane(IDane dane)
     }
 
     @Override
-    public void setHaslo(String pswd){}
+    public void setHaslo(String pswd){
+            try{
+                MessageDigest digest = MessageDigest.getInstance("SHA-256");
+                byte[] hash = digest.digest(pswd.getBytes("UTF-8"));
+                StringBuffer hexString = new StringBuffer();
+
+                for (int i = 0; i < hash.length; i++) {
+                    String hex = Integer.toHexString(0xff & hash[i]);
+                    if(hex.length() == 1) hexString.append('0');
+                    hexString.append(hex);
+                }
+
+                this.haslo = hexString.toString();
+            } catch(Exception ex){
+                throw new RuntimeException(ex);
+            }
+
+    }
 
```

Comment:
Jak możesz to wyciągnij haszowanie do osobnej funkcji bo się przyda na potem w innych klasach.